android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java

```diff
@@ -110,7 +110,7 @@ public void startMeeting(
       params.userId = userId;
       params.userType = userType;
       params.zoomAccessToken = zoomAccessToken;
-      params.zoomToken = zoomToken;
+//       params.zoomToken = zoomToken;
```

Comment:
could you comment on this?

Reply:
The commented line gives an error in build. I didn't look deeply but from initial look it seems that the parameter has been removed. The API docs specify that ZAK token is enough to start a meeting, and also zoomToken was previously not required to begin with.
I have tested the build after commenting it, yes I didn't update the readme and method params yet. Will you do that or I should take a look into that?

Reply:
got it, thnx. A lot has changed in zoom api since I last time updated the lib.
if you feel like updating the readme that would be great. I can probably take a look on that but later.

Btw regarding macos I am still waiting for zoom to support xcode 11 before proceeding.
But it is taking them so long..
![image](https://user-images.githubusercontent.com/3238370/89875201-40c25180-dbbd-11ea-8b16-dca9081f07fd.png)

Reply:
Understood lol, feel free to close this PR and use its changes when xcode 11 support comes out and you're able to test both platforms. Ill be happy to pitch in then and test the builds too.